Dockerfile

```diff
@@ -22,7 +23,8 @@ RUN gradle --console=plain \
     :hub:shared:build \
     :hub:shared:test
 
-FROM gradle:6.7.0-jdk11 as build-app
+ARG REGISTRY_IMAGE_GRADLE=gradle:6.7.0-jdk11
```

Comment:
Does this ARG need defining again here?

Reply:
I just read the docs. ARGS go out of scope at the end of each build phase, so yes it is!

Reply:
Think so args go out of scope at the end of every build stage https://docs.docker.com/engine/reference/builder/#arg